koku/api/provider/models.py

```diff
@@ -280,7 +280,10 @@ class ProviderInfrastructureMap(models.Model):
     """
 
     infrastructure_type = models.CharField(
-        max_length=50, choices=Provider.CLOUD_PROVIDER_CHOICES
+        max_length=50,
+        choices=Provider.CLOUD_PROVIDER_CHOICES,
+        null=True,
+        blank=False
```

Comment:
`null=True` refers to the Database allowing NULL values.
`blank=False` refers to the Serializer accepting `' '` (empty string) values.

This tightens-up the table to ensure that we only have one null value instead of two.

Reply:
The default for null (which is False) is actually what we want here. Having a record in this table with a null value for infrastructure type will break things. Can you switch this back?

Reply:
I've removed the null.